src/styles/index.scss

```diff
@@ -1,3 +1,199 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
```

Comment:
Properties within the `body` selector should be ordered according to the checklist. Flex properties should come after box model properties and before typography properties. Consider reordering them to follow the best practices.